Dockerfile

```diff
@@ -98,3 +99,23 @@ ENV UBSAN_OPTIONS=print_stacktrace=1
 ENV ASAN_OPTIONS=detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1
 RUN ln -s /opt/rocm/llvm/bin/llvm-symbolizer /usr/bin/llvm-symbolizer
 
+#Setup HIPIFY and rocRAND libraries (used for Onnxruntime)
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
+    hipfft \
+    rocthrust \
+    rocrand \
+    hipsparse \
+    rccl \
+    rccl-dev \
+    rocm-smi-lib \
+    roctracer-dev \
+    hipcub  \
+    hipblas  \
+    hipify-clang \
```

Comment:
Why do we need hipify?

Reply:
To build onnxrt. last build failed since it was looking for a hip compiler for the MIGraphX EP